### ISSUE TITLE
Custom languages not detected by silicon but detected by bat

### ISSUE THREAD

**Issue by jviotti** [Aloxaf/silicon#135]
Hey there! I'm running the latest `silicon` version from Homebrew. 

My `bat` config directory is:

```sh
$ bat --config-dir
/Users/jviotti/.config/bat
```

I placed various files inside the `syntaxes` directory:

```sh
$ ls /Users/jviotti/.config/bat/syntaxes
asn.sublime-syntax        capnproto.sublime-syntax  thrift.sublime-syntax
```

Which `bat` seems to detect correctly:

```sh
$ bat --list-languages
...
ASN                               asn
...
Cap'n Proto                       capnp
...
Thrift                            thrift
```

However `silicon` fails when trying to render a file with one of those languages:

```sh
$ silicon schema.thrift -o out.png --background '#ffffff' \
                --no-window-controls --no-line-number --theme GitHub --language thrift \
                --pad-horiz 20 --pad-vert 20
[error] Unsupported language: thrift
```

Am I missing something?

**Comment by jviotti** [comment]
My versions are:

```
$ bat --version
bat 0.17.1
$ silicon --version
silicon 0.3.2
```

**Comment by jviotti** [comment]
I just tried setting `BAT_CACHE_PATH` as per https://github.com/Aloxaf/silicon/blob/master/src/utils.rs#L16, however same thing:

```sh
$ BAT_CACHE_PATH="/Users/jviotti/.cache/bat" silicon schema.thrift -o out.png --background '#ffffff' \
                --no-window-controls --no-line-number --theme GitHub --language thrift \
                --pad-horiz 20 --pad-vert 20
[error] Unsupported language: thrift

$ ls /Users/jviotti/.cache/bat
metadata.yaml  syntaxes.bin   themes.bin
```

I did run `bat cache --build` in advance, so the syntaxes should be in the cache.

**Comment by jviotti** [comment]
OK, I have not dug into what's going on, but the latest version compiled from `master` works fine (while the Homebrew one doesn't)

**Comment by Aloxaf** [comment]
Closed by 0.4.0
